src/common/transformations/tests/op_conversions/convert_maxpool_downgrade_test.cpp

```diff
@@ -6,37 +6,174 @@
 
 #include <gtest/gtest.h>
 
-#include <memory>
-#include <string>
-
 #include "common_test_utils/ov_test_utils.hpp"
-#include "openvino/core/model.hpp"
+#include "openvino/op/max_pool.hpp"
 #include "openvino/opsets/opset1.hpp"
+#include "openvino/opsets/opset14.hpp"
 #include "openvino/opsets/opset8.hpp"
```

Comment:
Avoid including whole opsets. It causes significant object file size bloat, leading to compilation termination in some cases. Including only used ops should be enough.